src/pip/_internal/operations/install/wheel.py

```diff
@@ -609,19 +609,17 @@ def pyc_output_path(path: str) -> str:
 
     # Compile all of the pyc files for the installed files
     if pycompile:
-        with captured_stdout() as stdout:
-            with warnings.catch_warnings():
-                warnings.filterwarnings("ignore")
-                for path in pyc_source_file_paths():
-                    success = compileall.compile_file(path, force=True, quiet=True)
-                    if success:
-                        pyc_path = pyc_output_path(path)
-                        assert os.path.exists(pyc_path)
-                        pyc_record_path = cast(
-                            "RecordPath", pyc_path.replace(os.path.sep, "/")
-                        )
-                        record_installed(pyc_record_path, pyc_path)
-        logger.debug(stdout.getvalue())
```

Comment:
This change means that compilation errors are not displayed in the debug output anymore. I personally never had to look for that, but I just wanted to call it out and make sure this is intended.

Reply:
> The logic currently passes quiet=True but changing that to quiet=2 should silence all the output from that call.

Reply:
From https://docs.python.org/3/library/compileall.html#compileall.compile_file

> If quiet is False or 0 (the default), the filenames and other information are printed to standard out. Set to 1, only errors are printed. Set to 2, all output is suppressed.

The corresponding implementation: https://github.com/python/cpython/blob/c481cd6256f0e4493fa28f26c5902101803abadb/Lib/compileall.py#L250-L268

Reply:
Yep yep. But before this PR we were capturing the errors (quite=1) and logging them with debug level. This PR does not do that anymore. Unless I'm missing something (I'm a bit in covid brain fog atm) ?